benchmarks/benches/complicated_maud.rs

```diff
@@ -0,0 +1,102 @@
+#![feature(plugin, test)]
+#![plugin(maud_macros)]
+
+extern crate maud;
+extern crate test;
+
+use maud::{Markup, PreEscaped};
+
+#[derive(Debug)]
+struct Entry {
+    name: &'static str,
+    score: u16,
```

Comment:
I've opted for an usize since space is not an issue (and it will be aligned anyway) I thought that letting the compiler decide on the right thing (tm) might be the best